lib/pkgcloud/azure-v2/scripts/sudo.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Why add this file if it is not used? Did you mean to copy the original from DarwinJS/ChocoPackages?